CHANGELOG.md

```diff
@@ -4,7 +4,32 @@
 breaking changes in the upcoming 3.x release. This release is scheduled for
 2024-12 or 2025-01.
 
-## v2.17.0 - TBD
+## v2.17.0
+
+### [Compute Engine](/google/cloud/compute/README.md)
+
+- fix(compute): usable in external Bazel projects
```

Comment:
I think we should include #12877 in this release.

Reply:
Done. I sent the PR optimistically.